docs/devel/release.md

```diff
@@ -0,0 +1,64 @@
+# runtime-tools release guide
+
+## Release process
+
+This section shows how to perform a release of runtime-tools.
+The following example assumes we're going from version 0.5.0 (`v0.5.0`) to 0.6.0 (`v0.6.0`).
+
+Let's get started:
+
+- Start at the relevant milestone on GitHub (e.g. https://github.com/opencontainers/runtime-tools/milestones/v0.6.0): ensure all referenced issues are closed (or moved elsewhere, if they're not done). Close the milestone.
+- runtime-tools does not use a [roadmap file](https://github.com/opencontainers/runtime-tools/issues/465) but GitHub milestones. Update the [other milestones](https://github.com/opencontainers/runtime-tools/milestones), if necessary
+- Branch from the latest master, make sure your git status is clean
+- Update the [VERSION](https://github.com/opencontainers/runtime-tools/blob/master/VERSION)
+- Update the [release notes][changelog].
+  Try to capture most of the salient changes since the last release, but don't go into unnecessary detail (better to link/reference the documentation wherever possible).
+
+Ensure the branch is correct:
+
+- Ensure the build is clean!
+  - `git clean -ffdx && make && make test` should work
+- Integration tests on CI should be green
+- Check the version of the binaries:
+  - Check `./oci-runtime-tool --version`
+  - Check `./runtimetest --version`
+
+Once everything is fine:
+
+- File a PR
+- Ensure the CI on the release PR is green
+- Send an email to the [mailing list][mailinglist] ([example for v0.5.0](https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/iuWpWUai4_I)) and get reviews from other [maintainers][maintainers].
+- Once the maintainers agree, merge the PR
```

Comment:
Should we land local copies of the governance docs that we can link here to expand on “agree”?  My last attempt at that was #274, but perhaps the maintainers have a different plan in mind for maintaining governance docs for this project.